### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by willhsandbox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # Hello World
+
+Hi
```
Comment (doggy8088) [inline README.md:3]: Good job!

Comment (willhsandbox) [reply README.md:3]: Thanks.

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 # Hello World
+
```
Comment (doggy8088) [inline README.md:2]: Remove this blank line.

**Review by doggy8088 — APPROVED** [review]
LGTM

**Pull request merged**